.github/workflows/codeql-analysis.yml

```diff
@@ -1,66 +1,41 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
 name: "CodeQL"
 
 on:
   push:
-    branches: [main]
+    branches: [ "main" ]
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [main]
+    branches: [ "main" ]
   schedule:
     - cron: '0 15 * * 3'
 
 jobs:
   analyze:
     name: Analyze
-    runs-on: ubuntu-22.04
+
+    runs-on: [ubuntu-latest]
```

Comment:
Not sure if we want to unpin Ubuntu and change the version behind the scenes, can distro version stay explicitly stated?